src/libraries/System.ComponentModel.Composition/System.ComponentModel.Composition.sln

```diff
@@ -39,8 +39,8 @@ Global
 		{396D6EBF-60BD-4DAF-8783-FB403E070A56}.Debug|Any CPU.Build.0 = netstandard2.0-Debug|Any CPU
 		{396D6EBF-60BD-4DAF-8783-FB403E070A56}.Release|Any CPU.ActiveCfg = netstandard2.0-Release|Any CPU
 		{396D6EBF-60BD-4DAF-8783-FB403E070A56}.Release|Any CPU.Build.0 = netstandard2.0-Release|Any CPU
-		{2D694AC8-A12F-4622-9405-74E20EC40C3A}.Debug|Any CPU.ActiveCfg = netcoreapp5.0-Debug|Any CPU
-		{2D694AC8-A12F-4622-9405-74E20EC40C3A}.Debug|Any CPU.Build.0 = netcoreapp5.0-Debug|Any CPU
+		{2D694AC8-A12F-4622-9405-74E20EC40C3A}.Debug|Any CPU.ActiveCfg = netcoreapp2.0-Debug|Any CPU
```

Comment:
It seems like this wasn't intentional, could you please revert?

Reply:
whoops, yes it is not intended, will revert